src/App.tsx

```diff
@@ -1,24 +1,145 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useState, useEffect, useMemo } from 'react';
+import classNames from 'classnames';
+import { AddTodo } from './сomponents/AddTodo';
+import { TodosFilter } from './сomponents/TodosFilter';
+import { TodoList } from './сomponents/TodoList';
+import * as todoService from './api/todos';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [filterStatus, setFilterStatus] = useState<Filter>(Filter.all);
+  const [todosIdInLoading, setTodosIdInLoading] = useState<number[]>([]);
+
+  const showError = (message: string) => {
+    setErrorMessage(message);
+    setTimeout(() => setErrorMessage(''), 3000);
+  };
+
+  useEffect(() => {
+    todoService.getTodos()
+      .then(setTodos)
+      .catch(() => showError('Unable to load a todos'));
+  }, []);
+
+  const createTodo = (newTitle: string) => {
+    setTempTodo({
+      id: 0,
+      userId: 11337,
```

Comment:
why do you need to set hardcode user id and id

Reply:
about userId it's ok
but how not hardcode set id = 0?